services/autorust/openapi/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2017 Doug Tangren
```

Comment:
Is this the right license?

Reply:
It is unmodified. Should I add `Copyright (c) 2020 Cameron Taggart` to it?

Reply:
I started a thread with our CELA reps about this.